http/client/src/main/java/org/eclipse/rdf4j/http/client/query/AbstractHTTPQuery.java

```diff
@@ -31,7 +31,7 @@
 
 	protected final String baseURI;
 
-	public AbstractHTTPQuery(SPARQLProtocolSession httpClient, QueryLanguage queryLanguage, String queryString,
+	protected AbstractHTTPQuery(SPARQLProtocolSession httpClient, QueryLanguage queryLanguage, String queryString,
```

Comment:
I haven't checked - does this influence how this is shown in the Javadoc?

Reply:
Indeed, it would have been unhandy if it stopped showing up in the javadoc, but it still does.